okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -202,8 +202,7 @@ class HttpLoggingInterceptor @JvmOverloads constructor(
         val buffer = Buffer()
         requestBody.writeTo(buffer)
 
-        val contentType = requestBody.contentType()
-        val charset: Charset = contentType?.charset(UTF_8) ?: UTF_8
+        val charset: Charset = requestBody.contentType().utf8Charset()
```

Comment:
And I'm curious why not pass Utf8 to the `charset`? It was as a param for `readString` for once.
```kotlin
    logger.log("")
    if (buffer.isProbablyUtf8()) {
      logger.log(buffer.readString(charset))
      logger.log("--> END ${request.method} (${requestBody.contentLength()}-byte body)")
    } else {
      logger.log(
          "--> END ${request.method} (binary ${requestBody.contentLength()}-byte body omitted)")
    }
```

Reply:
I mean,seems in these logics,`UTF_8` is the default charset, no need to do this `contentType()?.charset(UTF_8) ?: UTF_8`.